relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -314,15 +316,42 @@ pub fn extract_tags(
                 } else {
                     scrubbed
                 };
-                Url::parse(url).ok().and_then(|url| {
+                if let Some(domain) = Url::parse(url).ok().and_then(|url| {
                     url.domain().map(|d| {
                         let mut domain = d.to_lowercase();
                         if let Some(port) = url.port() {
                             domain = format!("{domain}:{port}");
                         }
                         domain
                     })
-                })
+                }) {
+                    Some(domain)
+                } else if let Some(server_host) = span
+                    .data
+                    .value()
+                    .and_then(|data| data.get("server.address"))
+                    .and_then(|value| value.as_str())
+                {
+                    let lowercase_host = server_host.to_lowercase();
+                    let mut parts = lowercase_host.split(':');
+
+                    let domain = parts.next().unwrap_or("");
+                    let port = parts.next().and_then(|s| s.parse::<u16>().ok());
+                    if let Some(url_scheme) = span
+                        .data
+                        .value()
+                        .and_then(|data| data.get("url.scheme"))
+                        .and_then(|value| value.as_str())
+                    {
+                        span_tags.insert(
+                            SpanTagKey::RawDomain,
```

Comment:
Do we want to set `RawDomain` also in the `if` branch (where domain is derived from the span description)?

Reply:
Its not required for the feature we are building, as we can grab it from the span description otherwise. so I'm personally okay to leave as is for now